pkg/storage/export/virt-exportserver/exportserver.go

```diff
@@ -205,6 +241,138 @@ func NewExportServer(config ExportServerConfig) service.Service {
 	return es
 }
 
+var getExpandedVM = func() *virtv1.VirtualMachine {
+	f, err := os.Open(vmManifestPath)
+	if err != nil {
+		log.Log.Reason(err).Info("Unable to load VM manifest data")
+		return nil
+	}
+	defer f.Close()
+	fileinfo, err := f.Stat()
+	if err != nil {
+		log.Log.Reason(err).Info("Unable to load VM manifest data")
+		return nil
+	}
+	buf := make([]byte, fileinfo.Size())
+	_, err = f.Read(buf)
+	if err != nil {
+		log.Log.Reason(err).Info("Unable to load VM manifest data")
+		return nil
+	}
+
+	vm := &virtv1.VirtualMachine{}
+	if err := json.Unmarshal(buf, vm); err != nil {
+		log.Log.Reason(err).Info("Unable to load VM manifest data")
+		return nil
+	}
+	// Blank out the status
+	vm.Status = virtv1.VirtualMachineStatus{}
```

Comment:
I think the exportserver should be dumb about the stuff it serves up.  the controller should be responsible for sanitizing everything.

Reply:
Done